bubble_sort/Python3/mergeSort.py

```diff
@@ -0,0 +1,16 @@
+def bubbleSort(arr):
+    n = len(arr)
+ 
+    # Traverse through all array elements
+    for i in range(n):
+ 
+        # Last i elements are already in place
+        for j in range(0, n-i-1):
+ 
+            # traverse the array from 0 to n-i-1
+            # Swap if the element found is greater
+            # than the next element
+            if arr[j] > arr[j+1] :
+                arr[j], arr[j+1] = arr[j+1], arr[j]
+ a=list(map(int, input().split()))
```

Comment:
E901 IndentationError: unindent does not match any outer indentation level

*Origin: PycodestyleBear (E901), Section: `all.autopep8`.*